src/device/libpixy_m4/inc/sdmmc.h

```diff
@@ -16,22 +16,35 @@
 #ifndef __SDMMC_H__
 #define __SDMMC_H__
 
-#include <stdint.h>
+#include "blobs.h"
+#include "pixytypes.h"
 
 #define SDMMC_HEADER_MAGIC    "MTTR"
 #define SDMMC_HEADER_VERSION  1
 
 typedef struct __attribute__((packed))
 {
-    uint32_t magic;
-    uint32_t version;
-    uint32_t boot_cnt;
-    uint16_t crc16;
+    uint32_t magic;               // A specific number to make sure the sd card is formatted correctly
+    uint32_t version;             // Version of header structure
+    uint32_t session_cnt;         // Current recording session
```

Comment:
What increments the session count? is it boot and/or start command via I2C?

Reply:
The original recording feature enabled recording at boot-up. With the change to enable via I2C, a recording session is no longer tied to a bootup because some flights might not enable the recording. So the session counter will increment on the first enable command via I2C.